packages/greencheck/tests/bootstrap.php

```diff
@@ -3,8 +3,19 @@
  * Bootstrap doctrine and configuration
  *
  */
+
+function getConfigFilePath ()
+    {
+        if (getenv('TGWF_CONFIG_FILE_PATH')) {
+            return getenv('TGWF_CONFIG_FILE_PATH');
+        }
+        return '/tests/config.yml';
+    }
+
 $rootDir = __DIR__ . '/..';
 
+$config_file_path = __DIR__ . getConfigFilePath();
```

Comment:
i would use $configFilePath as name to be consistent with other variable names.

Reply:
Cool, will do. Python variable naming habits… 😄